### ISSUE TITLE
val_accuracy is low

### ISSUE THREAD

**Issue by chaos0625** [keras-team/keras-tuner#167]
I run the tunabel_resnet_cifar10.py, the log show that the accuracy is high (about 0.98), but the val_accuracy is low (about 0.6). 

Epoch 50/50
45000/45000 [==========================] - 449s 10ms/samplr - loss: 0.0350  - accuracy: 0.9882 - val_loss: 2.6765 - val_accuracy: 0.5934

Is it a normal performance?

**Comment by themrzmaster** [comment]
you are probably overfitting

**Comment by omalleyt12** [comment]
@chaos0625 I agree with @themrzmaster, it looks like the model is overfitting to the training data

Common things to try would be using a smaller model and/or adding dropout

**Comment by chaos0625** [comment]
@themrzmaster @omalleyt12 thanks for your reply, I'll add dropout and try it again
